packages/hub/src/lib/parse-safetensors-metadata.ts

```diff
@@ -119,14 +119,20 @@ async function parseShardedIndex(
 	const indexResp = await downloadFile({
 		...params,
 		path,
+		range: [0, 10_000_000],
 	});
 
 	if (!indexResp) {
 		throw new SafetensorParseError(`Failed to parse file ${path}: failed to fetch safetensors index.`);
 	}
 
 	// no validation for now, we assume it's a valid IndexJson.
-	const index: SafetensorsIndexJson = await indexResp.json();
+	let index: SafetensorsIndexJson;
```

Comment:
remove the comment one line above then?

Reply:
It's still true we don't enforce any validation on the payload